net/crowdsec/files/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Why it was added? No need. It takes LICENSE file from downloaded tarball, which is extracted.

Reply:
Removed

Reply:
I wondered why did you added in the first place and if you tested it.

Reply:
A simple mistake !
Thinking it was forget at the first time, but have not think of it in the downloaded tar.
Just an error...